test-app/tests/integration/components/textarea-field-test.gts

```diff
@@ -55,6 +55,21 @@ module('Integration | Component | TextareaField', function (hooks) {
     );
   });
 
+  test('it renders with a hint and label block', async function (assert) {
+    await render(<template>
+      <TextareaField @label="Label" @hint="Hint text" data-textarea>
+        <:label>Extra label content</:label>
+        <:hint>Extra hint content</:hint>
+      </TextareaField>
+    </template>);
+
+    let hint = find('[data-hint]');
+    let label = find('[data-label]');
+
+    assert.dom(hint).hasText('Hint text Extra hint content');
+    assert.dom(label).hasText('Label Extra label content');
```

Comment:
nit: Instead of doing a `find` can these be the following to match the other tests above?

```suggestion
    assert.dom('[data-hint]').hasText('Hint text Extra hint content');
    assert.dom('[data-label]').hasText('Label Extra label content');
```